__tests__/nc-news.test.js

```diff
@@ -149,8 +149,52 @@ describe("200 - GET /api/articles/article_id/comments", () => {
     return request(app)
       .get("/api/articles/1/comments")
       .then((response) => {
-        const articles = response.body.articleComments;
-        expect(articles).toBeSortedBy("created_at", { descending: true });
+        const articleComments = response.body.articleComments;
+        expect(articleComments).toBeSortedBy("created_at", {
+          descending: true,
+        });
+      });
+  });
+});
+
+describe("201 - POST - /api/articles/:article_id/comments", () => {
+  it("should respond with a 201 response and add a comment to an article with the following properties: username, body. And should respond with the posted comment", () => {
+    const newCommentObj = {
+      username: "rogersop",
+      body: "do you like my body?",
+    };
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(newCommentObj)
+      .expect(201)
+      .then((response) => {
+        const postedComment = response.body.newComment;
+        postedComment.forEach((comment) => {
+          expect(comment).toEqual(
+            expect.objectContaining({
+              comment_id: expect.any(Number),
+              article_id: expect.any(Number),
+              body: expect.any(String),
+              author: expect.any(String),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+  it("should respond with a 400 Bad Request if the username trying to be posted does not currently exist within the users database", () => {
+    const newCommentObj = {
+      username: "checkymarcus",
+      body: "do you like my body?",
+    };
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(newCommentObj)
+      .expect(400)
+      .then((response) => {
+        const errorMsg = response.body.msg;
+        expect(errorMsg).toBe("Bad request");
```

Comment:
Good test! 

Some more tests you can add for this one - 
/api/articles/90000/comments - what would happen? 
/api/articles/not-a-number/comments ? 
what if you sent a comment with the wrong data type for body / missing properties in the object?